src/pages/components/accordion/style.mdx

```diff
@@ -56,25 +72,41 @@ There is no max-height for an open panel but an accordion may
 vertical space. The width of an accordion varies based on the content, layout,
 and page design. The icon used in the header is a `chevron`.
 
-| Element | Property       | px/rem   | Spacing token |
-| ------- | -------------- | -------- | ------------- |
-| Header  | height         | 40 / 2.5 | –             |
-| Item    | border-top     | 1        | –             |
-| Icon    | size           | 16 / 1   | –             |
-|         | padding-right  | 16 / 1   | `$spacing-05` |
-| Title   | margin-left    | 16 / 1   | `$spacing-05` |
-| Panel   | padding-right  | 25%      | –             |
-|         | padding-top    | 8 / 0.5  | `$spacing-03` |
-|         | padding-left   | 16 / 1   | `$spacing-05` |
-|         | padding-bottom | 24 / 1.5 | `$spacing-06` |
+| Element                  | Property       | px/rem   | Spacing token |
+| ------------------------ | -------------- | -------- | ------------- |
+| Header                   | height         | 40 / 2.5 | –             |
+| Item                     | border-top     | 1        | –             |
+| Icon                     | size           | 16 / 1   | –             |
+| Icon: default alignment  | padding-right  | 16 / 1   | `$spacing-05` |
+| Icon: flush alignment    | padding-right  | 0        | –             |
+| Title                    | margin-left    | 16 / 1   | `$spacing-05` |
```

Comment:
Does title needs a "default" and "flush" distinction too?

Reply:
yes!